cloudify_agent/installer/__init__.py

```diff
@@ -341,16 +341,18 @@ def create_custom_env_file_on_target(self, environment):
             return None
 
     def download(self, url, destination=None):
+        local_cert_file = self.cloudify_agent['local_rest_cert_file']
```

Comment:
i have now found the `installer.download` is used in places where an agent package isn't installed.
i think this method should get these as parameters where verify will be `True` by default and only in
https://github.com/cloudify-cosmo/cloudify-agent/blob/master/cloudify_agent/installer/__init__.py#L127
should we supply values derived from `cloudify_agent`